src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -40,6 +44,33 @@
 
 logger = logging.getLogger(__name__)
 
+# @TODO: Find a home for these
```

Comment:
Opening issue for TODO

Reply:
This isn't unique to Python at all, but there isn't a utility bucket. Maybe there should be?

Reply:
This isn't unique to Python at all, but there isn't a utility bucket. Maybe there should be?

Reply:
I've proposed `backend/common` as a place for things like this; discussing over on slack now :)

Reply:
For posterity: https://pantsbuild.slack.com/archives/C0D7TNJHL/p1641577800027600

Reply:
`core/util_rules` seems to be the answer :tada: